src/ImmediatePayment.java

```diff
@@ -1,3 +1,13 @@
 public class ImmediatePayment extends Payment{
     private boolean phoneConfirmation;
+
+    public ImmediatePayment(Account account, Order order, float total) {
+        super(account, order, total);
```

Comment:
should get and initialize phoneConfirmation?

Reply:
we dont use that in the switch case, so i guess it isn't necessary.

Reply:
we can initialize it for our use.